reflex/utils/dependency.py

```diff
@@ -0,0 +1,47 @@
+"""Building the app and initializing all prerequisites."""
+
+from __future__ import annotations
+
+import os
+import re
+import subprocess
+
+from reflex import constants
+from reflex.utils import console
+
+
+def generate_requirements():
+    """Generate a requirements.txt file based on the current environment."""
+    # Run the command and get the output
+    result = subprocess.run(
+        ["pipdeptree", "--warn", "silence"],
```

Comment:
```suggestion
        [sys.executable, "-m", "pipdeptree", "--warn", "silence"],
```

more reliable invocation for venvs